Codigo/MODULO_NPCs.bas

```diff
@@ -2216,3 +2222,13 @@ End Function
 Public Function GetHitBonus(ByRef Npc As t_Npc) As Integer
     GetHitBonus = Npc.Modifiers.HitBonus
 End Function
+
+'Defines bonus that heald the user when its healed with something, a spell potion anything
```

Comment:
Typo `heald`

I would rephrase as:
`Defines the healing bonus when using a potion or a spell`